# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mapped_hyph/Cargo.toml
# Commit: a1ab26abf3eb
# Full Hash: a1ab26abf3eb645e202d69dffea86398a0ea4563
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-09-30 21:45:29
# Regressor Bug: 1667192
# File Overlap Count: 6
# Description:
#   Bug 1667192 - patch 1.1 - Run `mach vendor rust` to update in-tree mapped_hyph source. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D91409
# ==============================================================================

diff -r 66b86f8b6682 -r a1ab26abf3eb third_party/rust/mapped_hyph/Cargo.toml
--- a/third_party/rust/mapped_hyph/Cargo.toml	Wed Sep 30 12:51:17 2020 +0000
+++ b/third_party/rust/mapped_hyph/Cargo.toml	Wed Sep 30 12:51:54 2020 +0000
@@ -1,7 +1,7 @@
 [package]
 name = "mapped_hyph"
 description = "Hyphenation using precompiled memory-mapped tables"
-version = "0.3.0"
+version = "0.4.0"
 authors = ["Jonathan Kew <jfkthame@gmail.com>"]
 license = "MIT/Apache-2.0"
 edition = "2018"